typescript/phoenix-sdk/examples/seatManager.ts

```diff
@@ -0,0 +1,172 @@
+  const claimSeatIx = getClaimSeatIx(marketAddress, traderKeypair.publicKey);
+  const tx = new Transaction().add(claimSeatIx);
+  const txId = await phoenix.connection.sendTransaction(tx, [traderKeypair], {
+    skipPreflight: true,
+  });
+  await phoenix.connection.confirmTransaction(txId, "confirmed");
+  console.log(
+    `Claim seat tx Link: https://beta.solscan.io/tx/${txId}?cluster=devnet`
+  );
+
+  // This refreshes the market state from the blockchain, so we can check that claiming the seat succeeded.
+  await phoenix.refreshMarket(marketAddress);
+  const marketData = phoenix.markets.get(marketAddress.toBase58())?.data;
+  const traderIndex = marketData?.traderPubkeyToTraderIndex.get(
+    traderKeypair.publicKey.toBase58()
+  );
+  assert.ok(traderIndex !== undefined);
+}
+
+// Evict a trader from the market state. You may need to evict a trader if the market trader state is at capacity.
+// Notes:
+// - Eviction only works when the market trader state is at capacity.
+// - Eviction only works for a given trader when that trader does not have any locked base or quote tokens.
+// - In most cases, you do not need to explicitly call evict seat. You can use the instruction creator function `confirmOrCreateClaimSeatIxs`, which will create an evict seat instruction for you, if needed.
+// - If you wish to manually evict a trader and are unsure of which trader can be evicted, you can use the `findTraderToEvict` helper function.
+export async function evictSeat() {
+  const endpoint = "https://api.devnet.solana.com";
+  const connection = new Connection(endpoint);
+
+  // Use your keypair in the place of the below signerKeypair example to instantiate the Client.
+  const signerKeypair = new Keypair();
+  const phoenix = await Phoenix.Client.create(
+    connection,
+    endpoint,
+    signerKeypair.publicKey
+  );
+  console.log("Signer pubkey: ", signerKeypair.publicKey.toBase58());
+  // Request a SOL airdrop to send the transaction in this example. Only needed, and will only work, on devnet.
+  await phoenix.connection.requestAirdrop(
+    signerKeypair.publicKey,
+    1_000_000_000
+  );
+
+  // This is a SOL / USDC market on devnet.
+  const marketAddress = new PublicKey(
+    "CS2H8nbAVVEUHWPF5extCSymqheQdkd4d7thik6eet9N"
+  );
+
+  // Here, we simulate a trader to evict.
+  const traderKeypair = new Keypair();
+  const marketData = phoenix.markets.get(marketAddress.toBase58())?.data;
+  if (!marketData)
+    throw new Error(
+      "Market data not found for market: " + marketAddress?.toBase58()
+    );
+
+  const evictSeatIx = getEvictSeatIx(
+    marketAddress,
+    marketData,
+    traderKeypair.publicKey,
+    signerKeypair.publicKey
+  );
+
+  const tx = new Transaction().add(evictSeatIx);
+  const txId = await phoenix.connection.sendTransaction(tx, [signerKeypair], {
+    skipPreflight: true,
+  });
+  console.log(
+    `Evict seat tx Link: https://beta.solscan.io/tx/${txId}?cluster=devnet`
+  );
```

Comment:
if i run this, does it work?

Reply:
The transaction succeeds because when someone tries to evict a trader when the market state is not full, PSM returns `Ok(())` and the log `"Permissionless evictions are only allowed when the market is full"`. I feel like this is still helpful illustration of the example and doesn't need to be commented out.